package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcloud-storage-upload",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Especially releases should only occur on master, never in other branches!

What happens, when there is another feature that also does a release in it’s branch?